extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -8,11 +8,11 @@ public function getDelivery() {
 		$this->loadScript(new ResourceLoaderAdEngineSourcePointCSDelivery());
 	}
 
-	public function getSourcePointMessaging() {
+	public function getspmsg() {
```

Comment:
Keep in mind cached pages which inline HTML will have:
```js
window._sp_.bootstrap('{{domain}}/api/v1/ARecoveryEngine/SourcePointMessaging');
```
Should we keep the old methods as aliases till 1-2 releases after releasing these changes?